emcache/client.py

```diff
@@ -626,6 +630,7 @@ async def create_client(
     ssl: bool = DEFAULT_SSL,
     ssl_verify: bool = DEFAULT_SSL_VERIFY,
     ssl_extra_ca: Optional[str] = None,
+    autodiscovery: Optional[float] = None,
```

Comment:
It specifies how frequently to check for changes. The `float` is in seconds. As the point is to update the node list if there's a change in the cluster.

Reply:
Regarding the second paragraph. It is true, that AWS provides it but I see it implemented in GCP as well so it starting to be a semi standard: https://cloud.google.com/memorystore/docs/memcached/about-auto-discovery

Reply:
@pfreixes I made the setting into boolean and also added poll interval (60s) and timeout (5s). I don't think there is a good reason to call it aws_autodiscovery, since looks like Google Cloud also adopted it, do you agree?